scim2-sdk-common/src/main/java/com/bettercloud/scim2/common/messages/PatchOperation.java

```diff
@@ -216,17 +218,33 @@ static final class RemoveOperation extends PatchOperation
      * @param path The path targeted by this patch operation.
      * @throws ScimException If a path is null.
      */
-    @JsonCreator
     private RemoveOperation(
-        @JsonProperty(value = "path", required = true) final Path path)
-        throws ScimException
+            @JsonProperty(value = "path", required = true) final Path path)
+            throws ScimException
     {
       super(path);
       if(path == null)
       {
+        throw BadRequestException.noTarget(
+                "path field must not be null for remove operations");
+      }
+    }
+
+    @JsonCreator
+    private RemoveOperation(@JsonProperty(value = "path", required = true) Path path,
+                            @JsonProperty(value = "value") final JsonNode value) throws ScimException {
+      super(path);
+      if(path == null) {
         throw BadRequestException.noTarget(
             "path field must not be null for remove operations");
       }
+
+      if (null != value && "members".equalsIgnoreCase(path.toString())) {
```

Comment:
It is possible to put this value into enum or constant ?

Reply:
:heavy_check_mark: